phpunit.xml

```diff
@@ -10,7 +10,7 @@
          beStrictAboutTodoAnnotatedTests="true"
          failOnRisky="true"
          failOnWarning="true"
-         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xml"
```

Comment:
This project uses various PHPUnit version. This makes the XML schema always up-to-date :+1:

Reply:
Can you send this as a separate PR + to all other extension repositories? Thank you.

Reply:
Sure :+1: